README.md

```diff
@@ -110,81 +138,222 @@ Then just create `pod` as usual with:
 
-```console
-$ go build -o kubewatch main.go
 ```
 
-You can also use the Makefile directly:
+# Configure
+
+Kubewatch supports `config` command for configuration. Config file will be saved at `$HOME/.kubewatch.yaml`
 
-```console
-$ make build
 ```
+$ kubewatch config -h
 
-### Building with Docker
+config command allows admin setup his own configuration for running kubewatch
 
-```console
-$ make docker-image
-$ docker images
-REPOSITORY          TAG                 IMAGE ID            CREATED              SIZE
-kubewatch           latest              919896d3cd90        3 minutes ago       27.9MB
+Usage:
+  kubewatch config [flags]
+  kubewatch config [command]
+
+Available Commands:
+  add         add webhook config to .kubewatch.yaml
+  view        view .kubewatch.yaml
+
+Flags:
+  -h, --help   help for config
+
+Use "kubewatch config [command] --help" for more information about a command.
 ```
+### Example:
 
-## Download kubewatch package
+### slack:
+
+- Create a [slack Bot](https://my.slack.com/services/new/bot)
+
+- Edit the Bot to customize its name, icon and retrieve the API token (it starts with `xoxb-`).
+
+- Invite the Bot into your channel by typing: `/join @name_of_your_bot` in the Slack message area.
+
+- Add Api token to kubewatch config using the following steps
+
+  ```console
+  $ kubewatch config add slack --channel <slack_channel> --token <slack_token>
+  ```
+  You have an altenative choice to set your SLACK token, channel via environment variables:
+
+  ```console
+  $ export KW_SLACK_TOKEN='XXXXXXXXXXXXXXXX'
+  $ export KW_SLACK_CHANNEL='#channel_name'
+  ```
+
+### flock:
+
+- Create a [flock bot](https://docs.flock.com/display/flockos/Bots).
+
+- Add flock webhook url to config using the following command.
+  ```console
+  $ kubewatch config add flock --url <flock_webhook_url>
+  ```
+  You have an altenative choice to set your FLOCK URL
+
+  ```console
+  $ export KW_FLOCK_URL='https://api.flock.com/hooks/sendMessage/XXXXXXXX'
+  ```
+
+## Viewing config
+To view the entire config file `.kubewatch.yaml` use the following command.
+```
+$ kubewatch config view
+Contents of .kubewatch.yaml
+
+handler:
+  slack:
+    token: xoxb-xxxxx-yyyy-zzz
+    channel: kube-watch
+  hipchat:
+    token: ""
+    room: ""
+    url: ""
+  mattermost:
+    channel: ""
+    url: ""
+    username: ""
+  flock:
+    url: ""
+  webhook:
+    url: ""
+resource:
+  deployment: false
+  replicationcontroller: false
+  replicaset: false
+  daemonset: false
+  services: false
+  pod: true
+  job: false
+  persistentvolume: false
+  namespace: false
+  secret: false
+  configmap: false
+  ingress: false
+namespace: ""
 
-```console
-$ go get -u github.com/bitnami-labs/kubewatch
 ```
 
-## Configuration
 
-Kubewatch supports `config` command for configuration. Config file will be saved at $HOME/.kubewatch.yaml
+## Resources
 
-### Configure slack
+To manage the resources being watched, use the following command.
```

Comment:
Suggest a small addition (besides being mentioned :

... use the following command, changes will be saved to `$HOME/.kubewatch.yaml`.

Reply:
updated